ember/package.json

```diff
@@ -84,6 +85,7 @@
     "edition": "octane"
   },
   "dependencies": {
+    "brotli-webpack-plugin": "^1.1.0",
     "ember-cli-code-coverage": "^2.0.0",
     "ember-focus-trap": "^1.0.1",
     "luxon": "^3.2.1"
```

Comment:
Move those to `devDependencies`. Since an app must be built to be used, all dependencies are dev dependencies.